mifospay/src/main/java/org/mifos/mobilewallet/mifospay/history/ui/HistoryFragment.java

```diff
@@ -177,5 +181,38 @@ public void showHistoryFetchingProgress() {
         vStateView.setVisibility(View.GONE);
         rvHistory.setVisibility(View.GONE);
         pbHistory.setVisibility(View.VISIBLE);
+        scrollDownButton.setVisibility(View.GONE);
```

Comment:
The button w not be visible when the history is still being fetched. It will only be displayed after the transaction history is being fetched. That's the reason I set it here. Anything you find wrong, do suggest it :)